app/controllers/nonsipp/unregulatedorconnectedbonds/NameOfBondsController.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.nonsipp.unregulatedorconnectedbonds
+
+import config.Refined.Max5000
+import controllers.actions.IdentifyAndRequireData
+import controllers.nonsipp.unregulatedorconnectedbonds.NameOfBondsController._
+import forms.TextFormProvider
+import models.Mode
+import models.SchemeId.Srn
+import navigation.Navigator
+import pages.nonsipp.unregulatedorconnectedbonds.NameOfBondsPage
+import play.api.data.Form
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.SaveService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import viewmodels.implicits._
+import viewmodels.models.{FormPageViewModel, TextAreaViewModel}
+import views.html.TextAreaView
```

Comment:
Looks weird to me that this is a text area and not a text input. Asked design to double check.

Reply:
Design confirmed it should be text area.